Week1/arrays.js

```diff
@@ -0,0 +1,8 @@
+//append arrays
+let arraysInside = [];
+console.log ('my array will contain: eagle, horse and dog');
+console.log(arraysInside);
+let myFavoriteAnimals = ['eagle ', 'dog ', 'horse '];
+console.log(myFavoriteAnimals);
+arraysInside = ['baby pig ' , myFavoriteAnimals];
```

Comment:
Ah, this will result in an array inside an array, but it won't add 'baby pig' to the current array.

So this is not what the assignment was about, but it is very nice that you know how to do this!

Reply:
I added baby pig with .push